test/unit/server-settings/server-settings-store.ts

```diff
@@ -212,4 +212,49 @@ describe('ServerSettingsStore', () => {
       expect(ServerSettingsStore.getInstance().initialized).to.be.true;
     });
   });
+
+  describe('strictPosToken setting', () => {
+    beforeEach(async () => {
+      // Ensure clean state before each test
+      ServerSettingsStore.deleteInstance();
+      await ServerSetting.clear();
+    });
+
+    it('should initialize with default value false', async () => {
+      const store = ServerSettingsStore.getInstance();
+      await store.initialize();
+      
+      const strictPosToken = store.getSetting('strictPosToken');
+      expect(strictPosToken).to.equal(false);
+    });
```

Comment:
This test is fine, but aren't the other tests already covered by the generic method tests?